app/main.py

```diff
@@ -1,3 +1,67 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if type(other) == type(self):
```

Comment:
I think you should use isinstance() here, as in every method where you compare types. In any case, it will look more beautiful